src/modules/blocks/rsvp/advanced-options/template.js

```diff
@@ -33,10 +33,6 @@ class RSVPAdvancedOptions extends PureComponent {
 
 	getContent = () => (
```

Comment:
Are we using this AdvancedOptions component anywhere? If not, should we delete it?

Reply:
Kept it because of [this comment from Gustavo](https://theeventscalendar.atlassian.net/browse/ET-951?focusedCommentId=91917), and in case we wanted to bring it back again at some point. But I can totally delete it.

Reply:
OK fair. Either way, we have the history of the component so we can easily reimplement it if we remove it.